packages/vscode-mdx/syntaxes/mdx.tmLanguage.json

```diff
@@ -1,6 +1,6 @@
 {
-  "$schema": "https://raw.githubusercontent.com/Septh/tmlanguage/master/tmLanguage.schema.json",
-  "name": "Markdown React",
+  "$schema": "https://raw.githubusercontent.com/Septh/tmlanguage/master/tmlanguage.json",
+  "name": "MDX",
```

Comment:
I don’t know if / how this name is used, but most tmLanguage definitions seem to define it.

Reply:
Seems optional. And not explained in the grammar. But good to have it